src/Temporalio/Bridge/src/worker.rs

```diff
@@ -47,13 +46,51 @@ pub struct WorkerOptions {
     max_activities_per_second: f64,
     max_task_queue_activities_per_second: f64,
     graceful_shutdown_period_millis: u64,
-    max_concurrent_workflow_task_polls: u32,
+    workflow_task_poller_behavior: PollerBehavior,
     nonsticky_to_sticky_poll_ratio: f32,
-    max_concurrent_activity_task_polls: u32,
+    activity_task_poller_behavior: PollerBehavior,
     nondeterminism_as_workflow_fail: bool,
     nondeterminism_as_workflow_fail_for_types: ByteArrayRefArray,
 }
 
+#[repr(C)]
+pub struct PollerSimpleMaximum {
+    pub simple_maximum: usize,
+}
+
+#[repr(C)]
+pub struct PollerAutoscaling {
```

Comment:
```suggestion
pub struct PollerBehaviorSimpleMaximum {
    pub simple_maximum: usize,
}

#[repr(C)]
pub struct PollerBehaviorAutoscaling {
```

Pedantic, but figured the full `PollerBehavior` prefix makes more sense in the C header when using the C API